Add keycap selection post (#5)
keebio/keebio-blog: posts/2022-09-05-keycap-selection/index.mdx

---
slug: keycap-selection-guide
title: Keycap Selection Guide
authors: [keebio]
tags: [keyboards, keycaps]
image: posts/01.1_blue_iris.JPG
---

import useBaseUrl from "@docusaurus/useBaseUrl";
import Caption from "@site/src/components/Caption";

<img src={useBaseUrl('posts/01.1_blue_iris.JPG')} />
<p />

They come in shades of every kind. Some are fandom inspired and some give your keyboard just the right sound you want. Some you can order right away and others you'll have to wait for what feels like an eternity for the magic to arrive...

That's right, we're talkin' keycaps today! There are times when the Keebio crew will get some questions about what keycaps are recommended or what keyset will work on our boards.

<!--truncate-->

Today is the day... your questions get answered!

![](00_keycaps_focus.png)
<Caption text="Some pastels for you?" />

> 
> They come in shades of every kind. Some are fandom inspired and some give your keyboard just the right sound that you want. Some you can order right away and others you’ll have to wait for what feels like an eternity for the magic to arrive…
> 
> That’s right, we’re talkin’ keycaps today! There are times when the Keebio crew will get some questions about what keycaps are recommended or what keyset will work on our boards. 
> 
> Today is the day… your questions get answered!

Reviewer: Remove duplicated lines
```suggestion
```

## Iris
Here's the great news: you can easily cover the [Iris](https://keeb.io/collections/iris-split-ergonomic-keyboard) with a 104 keyset! Tah-daaah!

![](01_104_standard_keyset.png)

> <Caption text="Here's your typical 104 keyset. It includes your alphas, modifiers, function row, navigation keys and numpad."/>

Reviewer: ```suggestion
<Caption text="Here's your typical 104 keyset. It includes your alphas, modifiers, function row, navigation keys, and numpad."/>
```

The Iris depends heavily on the 1u size keycaps, and only two 2u keycaps are possible (on the Rev. 7), so it leaves folks with a few questions about how to cover all those switches.

> ![](02_KLE_Iris.png)

Reviewer: Update KLE from Colemak to QWERTY as discussed

<Caption text="Ah, the beauty of the Iris in its columnar stagger."/>

> The pro tip to remember is that you can take spare keycaps from the numpad, arrow keycaps or navigation keycaps. Specifically if you need keycaps for your 2u formatted Iris, you could snag the tab, capslock, and shift. The zero and enter keycaps from the numpad work well, too. On the other hand, if you’re not keen on having legends that don’t quite match function, novelties are a fun way to get around that. 

Reviewer: ```suggestion
The pro tip to remember is that you can take spare keycaps from the numpad, arrow keycaps, or navigation keycaps. For example, if you need keycaps for 2u thumb keys, you could snag the `0` and `Enter` keycaps from the numpad work well, too. On the other hand, if you’re not keen on having legends that don’t quite match function, novelties are a fun way to get around that, or you can get an [ortho kit](https://keeb.io/products/gmk-n9-ortholinear-add-on-kit).
```

![](01.1_blue_iris.JPG)

> <Caption text="You know it's a cool board if it matches your 3D printed cat."/>

Reviewer: ```suggestion
<Caption text="You know it's a cool board if it matches your 3D-printed cat."/>
```

When I was chatting with Danny about these options, he had this great point: “Since it’s just plastic, it doesn’t control what the keyboard does.” 

**You** get to decide what all your keycaps do! It’s really a matter of choosing what look you want.*

## Quefrency

> The [Quefrency](https://keeb.io/collections/quefrency-split-staggered-65-keyboard) is a 65XT split keyboard, so it’s critical to be aware of how to cover your split spacebar if this model is your jam. (Could be strawberry jam or raspberry jam--we’re a no judgement zone!) 

Reviewer: ```suggestion
The [Quefrency](https://keeb.io/collections/quefrency-split-staggered-65-keyboard) is a 65XT split keyboard, so it’s critical to be aware of how to cover your split spacebar if this model is your jam. (Could be strawberry jam or raspberry jam--we’re a no judgment zone!) 
```

![](03_KLE_QF.png)

> For example, with the Rev.4, the split spaces and the shift on the right side will be the keycaps that may need some extra thought put into them.

Reviewer: ```suggestion
For example, with the Rev. 4, the split spaces and the Shift on the right side will be the keycaps that may need some extra thought put into them.
```

> If we’re looking at the Rev.2 for example, you’d need to decide if you need some extra 1.25u, 1.5u, or 2.25u keycaps. You’ll want to look at the Keyboard Layout Editor website to help you decide what keycaps you’d like to have on your board.

Reviewer: ```suggestion
If we’re looking at the Rev. 2 for example, you’d need to decide if you need some extra 1.25u, 1.5u, or 2.25u keycaps. You’ll want to look at the Keyboard Layout Editor website to help you decide what keycaps you’d like to have on your board.
```

### Some small pointers for KLE
Yes, that [website](http://www.keyboard-layout-editor.com/) has an acronym! I was honestly confused the first time I learned about it, so here’s your notification to let you know about it. :) 

The ‘c’ annotation means convex (curved outward if that is confusing!), and is more suitable for a space keycap. 

Sometimes you’ll want to go for a spacebar kit, but this can go by a rotation of different names depending on the website you’re looking at. 

### Base Kit vs 40s Kit

> A 40s kit is a good start for 1.75 and 2.25 extras. On that note, base kits often include 1.75 and 2.25us. Base kits are usually alphas and modifiers, numpads are not included. What is included in a base kit can sometimes be different depending on the keycap shop, so be sure you check the details of your order before your final purchase.

Reviewer: ```suggestion
A 40s kit is a good start for 1.75u and 2.25u extras. On that note, base kits often include 1.75u and 2.25u keys. Base kits are usually alphas and modifiers, numpads are not included. What is included in a base kit can sometimes be different depending on the keycap shop, so be sure you check the details of your order before your final purchase.
```

Here at Keebio, we recommend NovelKeys and CannonKeys because they usually sell the entire set needed to cover your Quefrency board. They both have readily available keycaps as well as group buys, depending on what you fancy.

> ![](04_Cannon_Caps_9009_dark.png)

Reviewer: ![image](https://user-images.githubusercontent.com/204212/188984250-c8d2a300-5796-4c89-95c0-7c01dc4c77c8.png)
Add annotations to keyset picture to point out the special keycaps that the Quefrency needs for the split spacebar and split right Shift, and mark notes in caption and post.

One such set is the [Cannon Caps 9009 Dark](https://cannonkeys.com/products/cannoncaps-9009-dark). The 2.25u as well as split shift keycaps are available in this set. For the macros, you can use the function row that is provided, as a creative choice. 

As a side note, 1.5u and 1u kits will not always be included in a base set. Cannon and NovelKeys will usually have these, but keycaps off Amazon are not always standard. Be sure to check your details. :)

If it feels annoying to use keycaps that say a function that it isn’t for, you can dig into making your own keycap stickers or buying a pre-made keycap sticker set. This can be a great option if you want to fully customize the look of your keycaps. Getting a blank set and some stickers might be a fun way to do things!

## Sinc 

> The [Sinc](https://keeb.io/collections/sinc) is a 75% split keyboard, and needs you to get creative! You can easily steal keycaps from your numpad and different novelties or blanks. 

Reviewer: ```suggestion
The [Sinc](https://keeb.io/collections/sinc) is a 75% split keyboard, and needs you to get creative! You can easily steal keycaps from your numpad and different novelties or blanks to fill the left macro section of the board.
```

![](05_KLE_sinc.png)

If you search for a standard 104 keycap set, you’ll find a kit that will cover your entire board, save for the spacebar keycaps like with the Quefrency.

![](06_GMK_Serika_2_set.png)

A good example to look at is the [GMK SERIKA 2](https://novelkeys.com/products/gmk-serika-2). You can use keycaps from the numpad for the macro section. Or if you want to get fancy, the macro expansion will have some extra keycaps you can use. However, the Core Kit (aka Zambumon’s version of a “Base kit”) should have everything you need.

## Ortholinears

> We have a lovely collection of ortholinears here at Keebio: Nyquist, Viterbi, Levinson, BFO-9000, Dilly and DSP40.

Reviewer: Link each product

![](07_ortholinears.JPG)
<Caption text="The cali cats are as good at posing as the keebs are."/>

> 

Reviewer: Add info on being able to adapt a regular keyset like with the Iris

To get an ortholinear keycap set readily in stock, Drop is your best bet. Some Group Buys will also have ortholinear kits available from time to time. Being on the lookout will help you grab some cool ones.. 

> The MP3 series will have an ortho kit that fits most of our models as well.

Reviewer: ```suggestion
The MT3 series will have an ortho kit that fits most of our models as well.
```
